### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by onsd**
I'm curious about this project and found a typo while reading https://devprotocol.xyz/ja/faq
Therefore I fixed it

### CODE REVIEW

File: public/ja/faq/index.html [hunk 1]
```diff
@@ -574,7 +574,7 @@ <h4 class="panel-title">
 								<div id="question26" class="panel-collapse collapse" style="height: 0px;">
 									<div class="panel-body">
 										<p>
-											Dev protocolは持続かのな活動を支援しているので、将来的に開発する予定がないものに関しては申請が通らない場合があります
+											Dev protocolは持続可能な活動を支援しているので、将来的に開発する予定がないものに関しては申請が通らない場合があります
```
Comment (hhatto) [inline public/ja/faq/index.html:577]: This is not related to this change, but `Dev protocol` should be unified to `Dev Protocol`.

Comment (onsd) [reply public/ja/faq/index.html]: Thank you for suggesting! 
I'm going to fix it too :)

Comment (hhatto) [reply public/ja/faq/index.html]: Thanks!! 👍

**Review by hhatto — APPROVED** [review]
LGTM

**Pull request merged**